gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3724,8 +3731,8 @@ subroutine Run ( gc, import, export, clock, rc )
   ! swath only
    epoch_swath_regen_grid: do n=1,nlist
       call MAPL_TimerOn(GENSTATE,trim(list(n)%collection))
-      call MAPL_TimerOn(GENSTATE,"Swath regen")
       if (index(trim(list(n)%output_grid_label), 'SwathGrid') > 0) then
+         call MAPL_TimerOn(GENSTATE,"SwathRun")
```

Comment:
1.  I think the timer should be inside the alarm check
2. "Run" is redundante.
3. Maybe what you want is "Swath" as the outer timer and "destroy" and "create_bundle" subtimers